### ISSUE TITLE
Kann nicht speichern, wenn nicht vorher ein Speicherstand ausgewaehlt wurde

### ISSUE THREAD

**Issue by GWRon** [TVTower/TVTower#893]
aus dem Forum:
> Hi, ich habe mir die neuste Version runtergeladen und spiele die 64 Bit Version auf Ubuntu 22.04. Die neue Version macht einen tollen Eindruck - ich kann nur nicht speichern.
> Wenn ich einen Speicherstandnamen eingebe, kann ich danach nicht den "Speichern"-Button klicken. Er wird zwar gelb, aber sonst passiert nix.

Ich kann das hier nachvollziehen:
- Neues Spiel
- Speichern-Dialog aufrufen
- einen Namen eintippen der NICHT mit den Namen eines existierenden Speicherstands anfaengt
- Speichernbutton zwar "enabled" aber keine Reaktion

Sobald man einmal einen der Speicherstaende aktiviert hatte, funktioniert auch das Speichern mit "nigelnagelneuem Namen"

**Comment by GWRon** [comment]
```
	Method onClickSaveSavegame:Int( triggerEvent:TEventBase )
		Local button:TGUIButton = TGUIButton(triggerEvent.GetSender())
		If Not button Or button <> dialogueButtons[0] Then Return False
print "onClickSaveSavegame ... button ok. name=" +savegameName.GetValue()

		If Not SaveSavegame(savegameName.GetValue())
```
GetValue ist in dem Fall noch Leer... entsprechend bockt und blockt SaveSavegame.

**Comment by lonk0123** [comment]
Irgendwie scheint der Cursor noch verbuggt, irgendwie, wenn der Spielstand mit Ziffern zu tun hat:

https://user-images.githubusercontent.com/121659400/216618753-83d1a740-fbe1-403b-87d4-77fb331df6b1.mp4

**Comment by nittka** [comment]
Nein, der Coursor reagiert auf die Elemente im Hintergrund (bzw. behält seine Form im Moment des Öffnens eines Menüs) und wird nicht beim Öffnen eines Dialogs auf "normal" zurückgesetzt.

**Comment by lonk0123** [comment]
ouch, das ist aber unschön

**Comment by GWRon** [comment]
> Nein, der Coursor reagiert auf die Elemente im Hintergrund (bzw. behält seine Form im Moment des Öffnens eines Menüs) und wird nicht beim Öffnen eines Dialogs auf "normal" zurückgesetzt.

Eigener Issue und dort ueberlegen, wie wir das angehen sollten - eventuell muss der Cursor von jedem "Modalfenster" zurueckgesetzt werden.
